base/applications/rapps/appview.cpp

```diff
@@ -1359,16 +1411,18 @@ CAppsListView::SetDisplayAppType(APPLICATION_VIEW_TYPE AppType)
 
             /* Add columns to ListView */
             szText.LoadStringW(IDS_APP_NAME);
-            AddColumn(ColumnCount++, szText, 250, LVCFMT_LEFT);
+            AddColumn(ColumnCount++, szText, 368 /*250*/, LVCFMT_LEFT);
```

Comment:
That's what a version control system is for. Let's keep the heads sources clean. Rest of PR LGTM.
```suggestion
            AddColumn(ColumnCount++, szText, 368, LVCFMT_LEFT);
```